lib/contracts/code_generator.js

```diff
@@ -159,7 +160,11 @@ class CodeGenerator {
         web3Load = Templates.define_web3_simple({url: connection, done: 'done();'});
       } else {
         let connectionList = "[" + this.contractsConfig.dappConnection.map((x) => '"' + x + '"').join(',') + "]";
-        web3Load = Templates.web3_connector({connectionList: connectionList, done: 'done();'});
+        if (self.env === 'development') {
```

Comment:
Does "development" here mean using the --dev option? If so, you should use the isDev flag

Reply:
just development environment, it's better not to rely on the isDev option too much